src/main/java/racingcar/view/View.java

```diff
@@ -0,0 +1,98 @@
+package racingcar.view;
+
+import static racingcar.global.constants.MessageType.CAR_NAMES_REQUEST_MESSAGE;
+import static racingcar.global.constants.MessageType.TOTAL_COUNT_REQUEST_MESSAGE;
+import static racingcar.global.constants.NumberType.MIN_RACE_COUNT;
+import static racingcar.global.constants.SymbolType.CAR_NAME_DELIMITER;
+
+import camp.nextstep.edu.missionutils.Console;
+import racingcar.global.constants.MessageType;
+import racingcar.global.exception.ErrorMessage;
+import racingcar.global.exception.RaceException;
+
+public final class View {
+    public static String requestCarNames() {
+        printlnMessage(CAR_NAMES_REQUEST_MESSAGE);
+        String userInput = readLine();
+        Validator.validateInvalidDelimiter(userInput);
+        return userInput;
+    }
+
+    public static String requestCount() {
+        printlnMessage(TOTAL_COUNT_REQUEST_MESSAGE);
+        String userInput = readLine();
+        Validator.validateNumberInput(userInput);
+        Validator.validateRange(userInput);
+        return userInput;
+    }
+
+    public static void printlnMessage(final MessageType messageType) {
+        System.out.println(messageType.getMessage());
+    }
+
+    public static void printlnString(final String string) {
+        System.out.println(string);
+    }
+
+    private static String readLine() {
+        String userInput = Console.readLine();
+        Validator.validateNonBlankInput(userInput);
+        return userInput;
+    }
+
+    private static class Validator {
+        private static void validateNonBlankInput(final String input) {
+            if (input.isBlank()) {
+                throw RaceException.of(ErrorMessage.BLANK_INPUT_ERROR);
+            }
+        }
+
+        /* validate requestCarNames() */
+        private static void validateInvalidDelimiter(final String names) {
```

Comment:
```suggestion
        private static void validateDelimiter(final String names) {
```
정적 클래스를 선언하신 점 좋습니다. 하나로 묶는 것은 내포 클래스의 올바른 사용법이라 생각해요.
내포(nested) 클래스 중 정적 클래스는 내부(inner) 클래스가 아니라는 점 확인하시면 좋을 것 같아요. 대부분의 한글 교재에서 오역된 부분이에요.
https://docs.oracle.com/javase/specs/jls/se11/html/jls-8.html#jls-8.1.3
(An inner class is a nested class that is not explicitly or implicitly declared static.)

`Validator.validate~` 단어 중복은 피하고 싶은데 좋은 방법을 모르겠네요.

Reply:
아 .. ! 제가 내부 클래스라는 개념을 오용해서 사용하고 있었군요. 조금 더 자세히 알아보고 새로운 기술을 사용해보아야겠습니다.

추가로 내포 클래스 사용에 대해 긍정적인 견해를 내주셔서 감사합니다! 저는 오히려 `validate`라는 접두사를 사용하는 것이 통일성이 있다고 판단하였는데, 단어 중복이라는 단점이 있을 수 있겠군요.

해당 부분은 조금 더 고민해보고 다음 미션에 적용해보도록 하겠습니다. 감사합니다!